### DISCUSSION TITLE
How to position the quick fix list like the image

### DISCUSSION THREAD

**Discussion by std-enigma** [folke/trouble.nvim#467] [Q&A] [answered]
![240602_04h16m32s_screenshot](https://github.com/folke/trouble.nvim/assets/150489740/2a2e61ce-8bbd-452a-8ac2-b739d1af1942)
I want to use the trouble's qf list for searching with nvim-spectre 
and i'm clue less how to position it's window just like a regular qf list

NOTE: I also want to focus on it's window when it opens

**Comment by folke** [comment, marked as answer]
When you're in the spectre window: `:Trouble quickfix win.relative=win`

**Reply by std-enigma** [reply]
thanks folke love you a lot for all of your hard work
